org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/OurSyntaxDocument.java

```diff
@@ -358,19 +494,32 @@ private final int do_reapply(int comment, final String txt, final int line) {
 				}
 				setCharacterAttributes(oldi, i - oldi, styleString, false);
 			} else if (do_iden(c)) {
-				for (i++; i < n && do_iden(txt.charAt(i)); i++) {}
+				for (i++; i < n && do_iden(txt.charAt(i)); i++) {
+				}
 				AttributeSet style = (c >= '0' && c <= '9') ? styleNumber
 						: (do_keyword(txt, oldi, i - oldi) ? styleKeyword : styleNormal);
 				setCharacterAttributes(oldi, i - oldi, style, false);
 			} else {
 				for (i++; i < n && !do_iden(txt.charAt(i)) && txt.charAt(i) != '\n' && txt.charAt(i) != '-'
-						&& txt.charAt(i) != '/'; i++) {}
+						&& txt.charAt(i) != '/'; i++) {
+				}
 				setCharacterAttributes(oldi, i - oldi, styleSymbol, false);
 			}
 		}
 		return comment;
 	}
 
+	private boolean match(String txt, int i, String string) {
```

Comment:
Please use better argument names.  For example, rename `txt` to `sourceText`, `i` to `startIndex`, and `string` to `stringToMatch`

Reply:
These changes must be done more systematically. These were the variables in place.